Modules/Core/Common/test/itkMetaProgrammingLibraryTest.cxx

```diff
@@ -19,77 +19,72 @@
 #include <type_traits>
 
 #include "itkMetaProgrammingLibrary.h"
-#include "itkStaticAssert.h"
 
 
 int
 itkMetaProgrammingLibraryTest(int, char *[])
 {
   using namespace itk::mpl;
 
-#if !defined(ITK_LEGACY_REMOVE)
-
   // Or between constants
-  itkStaticAssert((OrC<true, true, true>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<true, false, true>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<true, true, false>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<false, false, true>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<true, false, false>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<false, true, false>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<false, false, false>::Value == false), "Unit test failed");
-
-  itkStaticAssert((OrC<true, true>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<true, false>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<false, true>::Value == true), "Unit test failed");
-  itkStaticAssert((OrC<false, false>::Value == false), "Unit test failed");
+  static_assert((OrC<true, true, true>::Value == true), "Unit test failed");
```

Comment:
Note that the extra pair of parentheses were necessary for the original `itkStaticAssert` macro calls, but are not needed for the static_assert's.

Reply:
Ah, I see now, you removed those parentheses already!